scripts/restore

```diff
@@ -77,7 +74,7 @@ ynh_script_progression --message="Restoring PHP-FPM configuration..."
 ynh_restore_file --origin_path="/etc/php/$phpversion/fpm/pool.d/$app.conf"
 
 # Recreate a dedicated PHP-FPM config
-ynh_add_fpm_config --usage=low --footprint=low --package="$extra_php_dependencies"
+ynh_add_fpm_config --usage=low --footprint=low
```

Comment:
(CI is failing because the apt dependencies should be installed before this line)